modules/installation-configuration-parameters.adoc

```diff
@@ -86,53 +50,19 @@ endif::[]
 ifeval::["{context}" == "installation-config-parameters-azure"]
 :azure:
 endif::[]
-ifeval::["{context}" == "installing-restricted-networks-azure-installer-provisioned"]
-:azure:
```

Comment:
Peer review

This `ifeval` is no longer required and is left over from https://github.com/openshift/openshift-docs/pull/61086. The Azure restricted installation assembly correctly references Installation configuration parameters for Azure.